### DISCUSSION TITLE
[R010] Anteprima attestazione conformità - Richiesta modifica servizio

### DISCUSSION THREAD

**Discussion by ManuelDeMeo** [italia/ansc#779] [Supporto servizi cooperativi] [answered]
Buon pomeriggio, faccio seguito alla issue #772 con una richiesta di modifica al servizio.

Come specificato qui: https://github.com/italia/ansc/issues/772#issuecomment-2007276779 , una volta firmata l'attestazione di conformità, per recuperare la stessa occorre consultare l'atto (R005), quindi esaminare la lista di allegati restituiti dal servizio e quindi, tramite R001, consultare l'attestazione come allegato.

Reputiamo che questo sia un overhead non indifferente e chiediamo che sia possibile consultare l'attestazione di conformità tramite servizio R010, anche una volta che la stessa risulti firmata.

**Comment by mttfranci** [comment, marked as answer]
Salve, l'R010 è solo  un servizio di anteprima e non recupera l'attestazione firmata.
Se fosse una soluzione accettabile stiamo valutando di restituire l'id dell'attestazione direttamente al momento della firma (che poi potrebbe essere recuperata sempre con il servizio R001 ma senza passare per la consultazione).

Saluti

Assistenza ANSC

**Comment by ManuelDeMeo** [comment]
> Salve, l'R010 è solo un servizio di anteprima e non recupera l'attestazione firmata. Se fosse una soluzione accettabile stiamo valutando di restituire l'id dell'attestazione direttamente al momento della firma (che poi potrebbe essere recuperata sempre con il servizio R001 ma senza passare per la consultazione).
> 
> Saluti
> 
> Assistenza ANSC

Buon pomeriggio,
sì, se non è possibile integrare l'R010, la proposta aiuterebbe evitando almeno l'invocazione di un servizio in più

Grazie della risposta
